dist/index.html

```diff
@@ -32,13 +32,9 @@
 		autoplay: true,
 		// set to true if you want to have video initially muted
 		mute: true,
-		captions: {
-			// set to true if you want video to have captions by default
-			enabled: true,
-			// custom CSS for captions, not all properties are available
-			// see: https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/#captions
-			styles: null
-		},
+		// set language for captions, must map captions' label, defaults to user browser language
+		// set to 'false' to turn them off completely
+		selectedCaptions: 'English',
```

Comment:
selectedCaptionsLanguage would align more with what it is, IMO -> selectedCaptions suggest there will be captions there 💃